Source/Core/Core/CoreParameter.cpp

```diff
@@ -373,6 +373,14 @@ void SCoreStartupParameter::CheckMemcardPath(std::string& memcardPath, std::stri
 	}
 }
 
+IVolume::ELanguage SCoreStartupParameter::GetCurrentLanguage(bool wii)
+{
+	if (wii)
+		return (IVolume::ELanguage)SConfig::GetInstance().m_SYSCONF->GetData<u8>("IPL.LNG");
```

Comment:
Something like that would be fine.  In practice, it's unlikely this will ever happen, given how SYSCONF is generated, but it's good style not to blindly cast to enum type without validating the value.